packages/logic-utils/src/utils/xml-node-to-ui5-node.ts

```diff
@@ -88,6 +92,37 @@ export function getUI5PropertyByXMLAttributeKey(
   return ui5Property;
 }
 
+export function getUI5NodeByXMLAttributeKey(
+  attribute: XMLAttribute,
+  model: UI5SemanticModel
+): UI5Prop | UI5Event | UI5Association | UI5Aggregation | undefined {
+  const parentElementClass = getUI5ClassByXMLElement(attribute.parent, model);
+  return parentElementClass
```

Comment:
can we have two separate return statements here?
one in case parentElementClass is undefined and one if it is not.

We may be doing too many things in a single expression which actually confused me the first time  I read it.